lib/features/manage_account/presentation/manage_account_dashboard_controller.dart

```diff
@@ -127,4 +129,12 @@ class ManageAccountDashBoardController extends ReloadableController {
       pushAndPopAll(AppRoutes.MAILBOX_DASHBOARD, arguments: sessionCurrent);
     }
   }
+
+  bool checkAvailableRuleFilterInSession() {
+    if(sessionCurrent?.capabilities.containsKey(capabilityRuleFilter) == true) {
+      EmailRulesBindings().dependencies();
```

Comment:
why not use `EmailRuleView` as a page, `dependencies` is controlled by `binding class`?